doc/getting_started/index.rst

```diff
@@ -24,6 +24,30 @@ Follow an operating system specific guide, then come back to this page.
    macOS <installation_mac.rst>
    Windows <installation_win.rst>
 
+.. python-pip::
+
+Python and pip
```

Comment:
From the commit log, if "(for example on Linux when installing CMake via pip3)" is important, why is this section placed after the linux specific document, instead of before?

Reply:
Good point, I will move it up. I think it doesn't hurt to have it at the top, since Zephyr relies so much on Python